posts/filters.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import filters
+from datetime import timedelta, datetime
+
+
+class StatusFilterBackend(filters.BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        return queryset.filter(**{"status": 1})
+
+
+class PeriodFilterBackend(filters.BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        if 'period' in request.query_params:
+            period = request.GET.get('min_lat')
+            if period == 'daily':
```

Comment:
Burada bir else de period'un gelmediği durumlar için gerekiyordu. Buradaki period optional bir parametre.